x-pack/test/api_integration/apis/telemetry/management_collection.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as ts from 'typescript';
+import * as path from 'path';
+import expect from '@kbn/expect';
+import { getInterfacesDescriptors, loadProgram } from '@kbn/telemetry-tools';
+import { FtrProviderContext } from '../../ftr_provider_context';
+// @ts-ignore
+
+export default function ({ getService }: FtrProviderContext) {
```

Comment:
Q: Do you think it might be easier to validate it against the schema exported in `src/plugins/kibana_usage_collection/server/collectors/management/schema.ts` instead of parsing and validating types?

Reply:
Yes! I'll update that. It did not cross my mind!